src/plot_erpgrid.jl

```diff
@@ -82,6 +82,26 @@ function plot_erpgrid!(
     hidedecorations!.(axlist)
     hidespines!.(axlist)
 
+    ax2 = Axis(f[1:8, 1:8]) #, xlabel = config.axis.xlabel, ylabel = config.axis.ylabel)
```

Comment:
is this for the label inset?

Why not use the same relative position as for the other axes?

Is there any check that it doesnt overlay with an actual axis?

Reply:
done